scripts/metadata_table.py

```diff
@@ -175,6 +176,19 @@ class MetadataHeader(ParseSource):
 
     __blank_line__ = re.compile(r"\s*[#;]")
 
+    __html_template__ = """
+<html>
+<head>
+<title>My First HTML</title>
```

Comment:
I obviously missed replacing this with a title containing the name of the metadata table ... stay tuned.